tests/e2e/crictl.go

```diff
@@ -52,19 +55,20 @@ func writeToFile(filename string, content string) error {
 }
 
 func crictlNewPodConfig(path string, name string) (string, error) {
-	podConfig := fmt.Sprintf(`{
-		"metadata": {
-			"name": "%s",
-			"namespace": "default",
-			"attempt": 1,
-			"uid": "abcshd83djaidwnduwk28bcsb"
+	podConfig := criruntimeapi.PodSandboxConfig{
+		Metadata: &criruntimeapi.PodSandboxMetadata{
+			Name:      name,
+			Uid:       "abcshd83djaidwnduwk28bcsb",
+			Namespace: "default",
+			Attempt:   1,
 		},
-		"linux": {
-		}
 	}
-	`, name)
+	psc, err := json.MarshalIndent(podConfig, "", "    ")
```

Comment:
is `psc` `podSandboxConfig`? maybe you could use this notation for clarity.

Reply:
It is just a temporary value to get the string from marshaling. It is in testing so I thought it was too big.